### PR TITLE
fix: transcribe UI subtitle were blocking chat

### LINKED ISSUE #16091: transcription block too long
cal video chat gets blocked if transcription is active

### PR DESCRIPTION
**Pull request by PeerRich**
image

### CODE REVIEW

**Review by Udit-takkar — COMMENTED** [review]
On mobile view It's blocking chat input box

Screenshot 2024-08-07 at 11 52 55 PM

**Comment by PeerRich** [comment]
> On mobile view It's blocking chat input box
> 
> Screenshot 2024-08-07 at 11 52 55 PM

ahh thats gonna be tricky to fix, cause right now the transcription is "outside" of the embed.

we could put it higher up but then it looks odd...

this PR is still better than before. maybe we just hide transcription on mobile

File: apps/web/modules/videos/views/videos-single-view.tsx [hunk 1]
```diff
@@ -95,7 +95,7 @@ export default function JoinCall(props: PageProps) {
       </Head>
       <DailyProvider callObject={daily}>
         <div
-          className="mx-auto"
+          className="mx-auto hidden sm:block"
```
Comment (PeerRich) [inline apps/web/modules/videos/views/videos-single-view.tsx:98]: @Udit-takkar this should hide transcription on mobile

**Review by Udit-takkar — APPROVED** [review]

**Comment by PeerRich** [comment]
a lot of workflow and webhook related E2E failing here 

CleanShot 2024-08-13 at 15 12 32@2x

**Pull request merged**